packages/manager/apps/telecom/src/components/telecom/telephony/sidebar/telephony-sidebar.service.js

```diff
@@ -63,54 +69,65 @@ angular.module('managerApp').service('TelephonySidebar', function TelephonySideb
           const sortedLines = tucVoipService
             .constructor.sortServicesByDisplayedName(billingAccount.getLines());
 
-          // display lines except plugAndFax and fax
-          const sortedSipLines = filter(
-            sortedLines,
-            line => !LINE_TYPES.ALL_FAX
-              .concat(LINE_TYPES.CARRIER_SIP)
-              .includes(line.featureType),
+          const groupedLines = tucVoipService.constructor.groupByFeatureType(sortedLines);
+
+          const sortedSipLines = get(
+            groupedLines,
+            TUC_TELEPHONY_VOIP_SERVICE_FEATURE_TYPES.DEFAULT_GROUP,
           );
 
           // add line subsections to billingAccount subsection
           const sipTrunkPrefix = $translate.instant('telecom_sidebar_section_telephony_trunk');
           const sipPrefix = $translate.instant('telecom_sidebar_section_telephony_line');
 
-          addServiceMenuItems(sortedSipLines, {
-            state: 'telecom.telephony.line',
-            prefix(lineService) {
-              return lineService.isSipTrunk() ? sipTrunkPrefix : sipPrefix;
-            },
-          }, billingAccountSubSection);
+          if (!isEmpty(sortedSipLines)) {
+            addServiceMenuItems(sortedSipLines, {
+              state: 'telecom.telephony.line',
+              prefix(lineService) {
+                return lineService.isSipTrunk() ? sipTrunkPrefix : sipPrefix;
+              },
+            }, billingAccountSubSection);
+          }
 
           // second get plugAndFax
-          const sortedPlugAndFaxLines = tucVoipService.constructor
-            .filterPlugAndFaxServices(sortedLines);
+          const sortedPlugAndFaxLines = get(
+            groupedLines,
+            TUC_TELEPHONY_VOIP_SERVICE_FEATURE_TYPES.GROUPS.plugAndFax,
+          );
 
           // add plugAndFax subsections to billingAccount subsection
-          addServiceMenuItems(sortedPlugAndFaxLines, {
-            state: 'telecom.telephony.line',
-            prefix: $translate.instant('telecom_sidebar_section_telephony_plug_fax'),
-          }, billingAccountSubSection);
+          if (!isEmpty(sortedPlugAndFaxLines)) {
+            addServiceMenuItems(sortedPlugAndFaxLines, {
+              state: 'telecom.telephony.line',
+              prefix: $translate.instant('telecom_sidebar_section_telephony_plug_fax'),
+            }, billingAccountSubSection);
+          }
 
           // then get fax
-          const sortedFaxLines = tucVoipService.constructor.filterFaxServices(sortedLines);
+          const sortedFaxLines = get(
+            groupedLines,
+            TUC_TELEPHONY_VOIP_SERVICE_FEATURE_TYPES.GROUPS.fax,
+          );
```

Comment:
```suggestion
          [],
          );
```
Don't know if it's safer or not

Reply:
Not needed because of the if condition under : https://github.com/ovh-ux/manager/pull/1499/files#diff-e3dadd8546046ad7b52489f7ec33587dR113 ;)

Reply:
`isEmpty(undefined) === true`